Sources/BugsnagPerformance/Private/BugsnagPerformanceImpl.h

```diff
@@ -124,8 +124,8 @@ class BugsnagPerformanceImpl: public PhasedStartup {
     void uploadPValueRequest() noexcept;
     void uploadPackage(std::unique_ptr<OtlpPackage> package, bool isRetry) noexcept;
     void possiblyMakeSpanCurrent(BugsnagPerformanceSpan *span, SpanOptions &options);
-    std::unique_ptr<std::vector<std::shared_ptr<SpanData>>>
-      sendableSpans(std::unique_ptr<std::vector<std::shared_ptr<SpanData>>> spans) noexcept;
+    NSMutableArray<BugsnagPerformanceSpan *> *
```

Comment:
Replacing vectors with NSMutableArrays can cause a performance hit. How much - I'm not sure. I think we need to profile it and see if it's a problem or not.

Reply:
@tomlongridge we should probably get some measurements on this before we merge into next, to make sure we're not introducing a bad performance hit.

Reply:
Measurements showed a 5-50% improvement in performance, so that's a nice bonus :)